app.js

```diff
@@ -12,6 +13,8 @@ app.get("/api/reviews", getReviews);
 
 app.get("/api/reviews/:review_id", getReviewById);
 
+app.get("/api/reviews/:review_id/comments", getCommentsByID);
+
 app.use((err, req, res, next) => {
```

Comment:
You could potentially separate the error handlers into 2:
- one for psql errors
- one for custom errors.